### ISSUE TITLE
Appropriate disk size when using s3.

### ISSUE THREAD

**Issue by zdyj3170101136** [ClickHouse/ClickHouse#71851]
### Company or project name

jaeger

### Question

We separate hot and cold data, put hot data (one day) on disk, and cold data (more than one day) on S3, and configure the data to be moved from disk to S3 when the disk usage is greater than 90%.

So what is the appropriate disk size? We hope to minimize it to reduce costs.

Is one day's data enough?

**Comment by Algunenano** [comment]
It's impossible for someone to give you any good answer to this as it depends in many factors that only you can know, such as how data is ingested and updated, what's your infrastructure and how much you pay for each element of it, what are the performance and reliability requirements.

If you want to minimize cost you shouldn't use s3 and just buy physical, cheap and reliable sata disks, but it's likely that performance will be worse and you'll have to maintain them.

> So what is the appropriate disk size? We hope to minimize it to reduce costs.
> Is one day's data enough?

Basically you'll need to test things out in your requirement, see if it's enough and iterate.

My recommendation, set it to a day (or any other value), see how the system performs for a while, and then decide if you need to increase or decrease it.
